script.module.buggalo/README.md

```diff
@@ -0,0 +1,147 @@
+http://forum.xbmc.org/showthread.php?tid=121925&pid=1137307#pid1137307
+
+
+HOW TO USE
+==========
+To use this script you must do these things besides importing it.
+
+1.  Choose whether to submit the collected data to an Email account or
+    a private website containing buggalo-web.
+
+    *  To use an email account set the `buggalo.EMAIL_RECIPIENT` with a dict structure containing information about the recipient, the email server and the credentials.
+       ```python
+       buggalo.EMAIL_CONFIG = {"recipient":"youremail@gmail.com", 
+                        "sender":"Buggalo <buggalo_account@gmail.com>", # example
+                        "server":"smtp.googlemail.com", # example for gmail
+                        "method":"ssl",
+                        "user":"buggalo_account@gmail.com",
+                        "pass":"yourpasswordforbuggalo_account"}
+       ```
+       You should use encrypted communication with the email server via smtp. For this option, the `method` has to be `ssl` and you have to give the username (`user`) and the password (`pass`), as in the example above. These are publicly visible and should therefore be used only for the smtp server and for nothing else (like e.g. for account login). This means, you have to create an email account for your addon to send the bug report emails.  
+       Unencrypted login is also possible by setting the `method` to `default` and not giving the credentials. Most of the emails will be blocked by ISPs though for spam prevention.
+
+    *  To use a website, set buggalo.SUBMIT_URL to a full URL where the
+       collected data is submitted.
+
+2.  Surround the code you want to be covered by this script in a
+    try..except block, such as:
+
+    ```python
+    try
+        # addon logic
+    except Exception:
+        buggalo.onExceptionRaised()
+    ```
+
+    For plugin type addons, it is a good idea to include pretty much
+    everything inside the try..except block.
+    See this link for an example:
+    https://github.com/xbmc-danish-addons/plugin.video.news.tv2.dk/blob/master/addon.py#L124
+
+    For script type addons, besides the rule above, each event in
+    your UI should include the try..except block as well.
+    See this link for an example:
+    https://github.com/twinther/script.tvguide/blob/master/gui.py#L140
+
+    Optionally, instead of writing the try..except block yourself, you
+    can decorate the function with @buggalo_try_except(). Function
+    specific extra data may be provide in the decorator:
+
+    ```python
+    @buggalo_try_except({'class' : 'MyWindowXML', 'method' : 'onInit', 'other_key' : 'other_value'})
+    def onInit(self)
+        pass
+    ```
+
+3.  If you chose to use a private website in step 1, now is the time to configure that
+    - otherwise there is no step 3...
+
+    A good starting point for the website is my buggalo-web module on github:
+    https://github.com/twinther/buggalo-web
```

Comment:
I did not test the code referenced there. Since the interfaces of this addon did not change, I assume it still works. I only took over the abandoned project to make a few fixes and am not able to do major changes/refactorizations or deep checks on references.